src/test/suite/fixtures/recorded/languages/scala/takeClass.yml

```diff
@@ -0,0 +1,29 @@
+languageId: scala
```

Comment:
One quick thing: we generally prefer "clear" instead of "take" for these tests, as it makes the test cases easier to read

Reply:
done!